### PR TITLE
Fix traceformat help output

### PR DESCRIPTION
**Pull request by JasonFengJ9**
Fix traceformat help output

Replaced `java com.ibm.jvm.format.TraceFormat` with `traceformat`.

closes: #823 

Reviewer @pshipton 

Signed-off-by: Jason Feng <fengj@ca.ibm.com>

### CODE REVIEW

File: jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/format/TraceFormat.java [hunk 1]
```diff
@@ -86,7 +86,7 @@
 
 	// constants
 	protected static final String usageMessage = "Usage:\n"
-			+ "java com.ibm.jvm.format.TraceFormat input_filespec [output_filespec] \n"
+			+ "traceformat input_filespec [output_filespec] \n"
```
Comment (pshipton) [inline jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/format/TraceFormat.java]: We still need to support IBM builds of Java 8 which don't have the traceformat executable. This should use the preprocessor to select which output to use.

**Comment by JasonFengJ9** [comment]
Pull request has been updated, pls have another look.

File: jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/format/TraceFormat.java [hunk 2]
```diff
@@ -86,7 +86,11 @@
 
 	// constants
 	protected static final String usageMessage = "Usage:\n"
+/*[IF Sidecar18-SE]*/
```
Comment (pshipton) [inline jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/format/TraceFormat.java]: Please use the Sidecar18-SE-OpenJ9 flag so all OpenJ9 (Java 8, 9, etc) refer to the traceformat execuable.

Comment (JasonFengJ9) [reply jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/format/TraceFormat.java]: Fixed, pls review again.

**Pull request merged**